src/components/GameList/GameList.tsx

```diff
@@ -98,107 +97,63 @@ export class GameList extends React.PureComponent<GameListProps, GameListState>
 
     computeRemainingTime(
         time_control: JGOFTimeControl | undefined,
-        clock: AdHocClock,
-        player_clock: AdHocPlayerClock | number,
+        clock: JGOFClock,
+        player_clock: JGOFPlayerClock,
     ) {
+        if (clock?.start_mode) {
+            return clock.start_time_left || 0;
+        }
```

Comment:
Currently `start_time_left` is returned here regardless of whose turn it is. Is that reasonable? Or should it only be returned if it applies to the selected `player_clock`?